worker/src/unstract/worker/clients/test_docker.py

```diff
@@ -109,6 +109,7 @@ def test_get_image(docker_client, mocker):
 def test_get_container_run_config(docker_client, mocker):
     """Test the get_container_run_config method."""
     os.environ[Env.WORKFLOW_DATA_DIR] = "/source"
+    execution_data_folder_prefix = os.getenv(Env.EXECUTION_RUN_DATA_FOLDER_PREFIX)
```

Comment:
@Deepak-Kesavan  not like this. What I meant is set this env like in line number 111
```suggestion
    os.environ[Env.EXECUTION_RUN_DATA_FOLDER_PREFIX] = "/app/workflow_data"
```